### DISCUSSION TITLE
Finished a build: Thanks! Feedback

### DISCUSSION THREAD

**Discussion by darraghh1** [mwood77/winderoo#51] [Ideas]
Hey Author,

First, I'd like to thank you! Great work on the HW and SW. 

Things I would change:

- Pulley system, using timing pulleys and belts.
- Needs a full enclose to protect electronics
- Motor wiring: I would add a hole to pull the wiring through to the back of the system. 
- Mounting points for the hardware you list in the BOM. Easy to mount. 

Keep up the good work!

**Comment by mwood77** [comment]
Thanks!
